### DISCUSSION TITLE
What does "skip_updates" do?

### DISCUSSION THREAD

**Discussion by departet** [aiogram/aiogram#671] [Q&A] [answered]
Hey,

so from the example code in the last line i got
`executor.start_polling(dp, skip_updates=True)`
In the documentation about the dispatcher it says it's recommended to not do this when dealing with payments or having high workload. There it also says updates deliver messages, edits, poll results, etc etc.
But even with this parameter i receive the messages i send my bot. So what is skip_updates doing exactly and is there any reason to not just get rid of this parameter?

Thank you very much in advance,
a python newby

**Comment by Olegt0rr** [comment, marked as answer]
It skips all incoming updates before start listening new updates

**Reply by departet** [reply]
Does this mean if multiple people would msg the bot at the same time some of the messages  might get dropped and thats the reason it's not recommended to use on heavy work load?

**Reply by Olegt0rr** [reply]
ALL unread messages will be dropped on call
